### PR TITLE
BCP - rackspace-billing-faq

### PR DESCRIPTION
**Pull request by catlook**
placeholder PR

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-522cc2ea19/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-6cfd0a414d/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-c0b6137023/how-to/). :bow:

**Comment by cyrichardson** [comment]
@catlook Suggestions in 
[rackspace-billing-faq_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1721592/rackspace-billing-faq_cyrEdits.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4d79bfc76c/how-to/)

File: content/general/rackspace-billing-faq.md [hunk 1]
```diff
@@ -110,33 +108,46 @@ are added to your bill.
...
+7.  Check the check box at the bottom of the page.
+8.  Type the word "CANCEL" into the text box.
+9.  Click **Cancel Account**.
+
+Cancellations cannot be initiated over the phone or with a support ticket. The cancellation
+submission is your authorized approval for Rackspace to delete your account and
+cancel your services in their entirety.  If you do not 
+have a copy of your data outside of our Cloud infrastructure, Rackspace is not liable for data loss.
+
+**Note** Your Rackspace cloud account is billed one month in arrears on your anniversary date.
```
Comment (chri2547) [inline content/general/rackspace-billing-faq.md]: Need a colon after 'Note'.

Comment (catlook) [reply content/general/rackspace-billing-faq.md]: fair enough

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-037d0f70a9/how-to/)

**Pull request merged**